giles-eco/src/main/resources/velocitytemplates/email/accountRevoked.vm

```diff
@@ -0,0 +1,5 @@
+Hi $name:
+
+Your Giles account request has been revoked by the admin!
```

Comment:
Let's make this:
"Your Giles account as been revoked. If you think this was an error please contact a Giles administrator.